locales-src/asrouter.ftl

```diff
@@ -81,3 +81,29 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Recommended features
+
+cfr-feature-type-recommended = Recommended Feature
```

Comment:
I believe flod wanted a comment describing how this is used: https://github.com/mozilla/activity-stream/pull/5231#discussion_r311714464

These existing strings seem to be used the same way as headers:
https://github.com/mozilla/activity-stream/blob/3b224ad27ca1846d701d500d988f7ee8bfc151b9/locales-src/asrouter.ftl#L5-L6

So the string id probably should be more like the existing ones instead of some "feature type"

Reply:
Given we already have these strings, it would be a good idea to move this new one there (and change the ID), and add the comment as a section comment.

```
## These messages are used as headings in the recommendation doorhanger

cfr-doorhanger-extension-heading = Recommended Extension
cfr-doorhanger-feature-heading = Recommended Feature
cfr-doorhanger-pintab-heading = Try This: Pin Tab
##
```

The `##` after the strings resets the section comment.

Reply:
will do! :)